lib/pdf_fill/extras_generator.rb

```diff
@@ -82,28 +84,43 @@ def set_header(pdf)
       end
     end
 
-    def generate
-      folder = 'tmp/pdfs'
-      FileUtils.mkdir_p(folder)
-      file_path = "#{folder}/extras_#{SecureRandom.uuid}.pdf"
-      generate_blocks = sort_generate_blocks
+    def add_page_numbers(pdf)
+      pdf.number_pages 'Page <page>',
+                       start_count_at: @start_page,
+                       at: [pdf.bounds.right - 50, 0],
+                       align: :right,
+                       size: 9
```

Comment:
I think the style preference in vets-api is to include parentheses for method calls, so let's add those here.